src/components/MoveButton.vue

```diff
@@ -205,6 +205,8 @@ export default class MoveButton extends Vue {
       if (times && typeof times === "number") {
         // the \b is necessary for things like '1t-a3', so the 3 is not caught
         command = command.replace(/\b[0-9]+/g, x => ('' + (parseInt(x) * times)));
+        // this is used for bids. 
+        command = command.replace(/0+/g, x => ('' + times));
```

Comment:
This can potentially conflict and create a weird bug with something else in the future.

I would do this instaed, outside the if (after it):

```
command = command.replace(/\b\$times\b/g, times ?? 0);
```

And instead of:

```
command: `${Command.Bid} ${pos.faction} 0`,
```
Do
```
command: `${Command.Bid} ${pos.faction} $times`,
```